server/src/main/java/com/example/tyfserver/member/controller/MemberController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.tyfserver.member.controller;
+
+import com.example.tyfserver.auth.dto.LoginMember;
+import com.example.tyfserver.member.dto.MemberResponse;
+import com.example.tyfserver.member.dto.NicknameValidationRequest;
+import com.example.tyfserver.member.dto.PointResponse;
+import com.example.tyfserver.member.dto.UrlValidationRequest;
+import com.example.tyfserver.member.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/members")
+public class MemberController {
+
+    private final MemberService memberService;
+
+    @PostMapping("/validate/url-name")
+    public ResponseEntity<Void> validateUrlName(@Valid @RequestBody UrlValidationRequest request) {
+        memberService.validateUrl(request);
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping("/validate/nickname")
+    public ResponseEntity<Void> validateNickname(@Valid @RequestBody NicknameValidationRequest request) {
+        memberService.validateNickname(request);
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/{urlName}")
+    public ResponseEntity<MemberResponse> memberInfo(@PathVariable String urlName) {
+        return ResponseEntity.ok(memberService.findMember(urlName));
+    }
+
+    @GetMapping("/point")
```

Comment:
```suggestion
    @GetMapping("/me/point")
```

이전에 미션에서 토큰 발급받을 때 처럼, URI 경로에 `me`를 추가해서 자기 자신의 point를 조회한다는 의미를 명확히 해주면 어떨까?

Reply:
그렇다면 본인의 정보에 관한 모든 url에 ``me``를 넣어서 통일 시켜줘야 할 것 같은데, 이 부분은 좀 더 생각해봐야할 것 같어

Reply:
굿굿 더 명시적이고 좋군여